extensions/google-translate/src/actions.tsx

```diff
@@ -18,39 +18,25 @@ interface ActionsOpts {
   otherSections?: React.ReactElement[];
 }
 
-export function getActions({
-  value,
-  defaultActionsPrefix,
-  firstSectionTitle,
-  otherActions,
-  otherSections,
-}: ActionsOpts) {
+export const ConfigurableAction = ({ defaultActionsPrefix, value }: ActionsOpts) => {
   const defaultPreference = getPreferenceValues<Preferences>().defaultAction;
-  const action = [
+
+  if (defaultPreference === DefaultActionPreference.PasteToApp) {
+    return (
```

Comment:
With your changes it's not possible to have both paste and copy actions in the same actions panel. This is basically why I introduced my function. I wanted instead of returning the single action the ordering based on the preferable default action to be applied. Probably this function can return a sorted array but it's up to you. Thanks for the refactoring!

Reply:
ahhhh, yeah sorry I've misunderstood it

let me update the code

Reply:
done

Reply:
LGTM! You can merge it

Reply:
I have no ability to do it, button is just not shown to me 😄
<img width="998" alt="Screenshot 2024-04-30 at 13 35 26" src="https://github.com/Popalay/raycast-extensions/assets/13422799/5a287cbd-a4de-4418-989c-09d9636c87f6">

Reply:
Done